### ISSUE TITLE
Multiple uses of the same action results in file conflict

### ISSUE THREAD

**Issue by andsnpl** [easingthemes/ssh-deploy#164]
Same issue as #131: I am trying to run ssh-deploy in two steps, one to copy the program artifacts, and one to copy a systemd config that goes in a different location. Both steps have SCRIPT_AFTER, but when the second one tries to run, the script file already exists and is not overwritten, so whatever I've put in the second step doesn't execute.

**Comment by andsnpl** [comment]
Here is a minimal repro

```yml
name: CI

on: [push, pull_request]

jobs:
  deploy:
    name: Deploy
    runs-on: ubuntu-latest
    steps:
      - uses: DamianReeves/write-file-action@v1.2
        with:
          path: ./deployment/bin/myservice
          write-mode: overwrite
          contents: |
             #!/bin/sh
             echo hello world
      - uses: DamianReeves/write-file-action@v1.2
        with:
          path: ./deployment/systemd/myservice.service
          write-mode: overwrite
          contents: |
            [Unit]
            Description=Application server

            [Service]
            Type=simple
            ExecStart=/home/${{ secrets.DEPLOY_USER }}/bin/myservice

            [Install]
            WantedBy=multi-user.target
      - uses: easingthemes/ssh-deploy@v4
        with:
          SSH_PRIVATE_KEY: ${{ secrets.DEPLOY_SSH_KEY }}
          REMOTE_HOST: ${{ secrets.DEPLOY_IP_ADDRESS }}
          REMOTE_USER: ${{ secrets.DEPLOY_USER }}
          SOURCE: ./deployment/bin/
          TARGET: /home/${{ secrets.DEPLOY_USER }}/bin/
          SCRIPT_AFTER: |
            chmod +x /home/${{ secrets.DEPLOY_USER }}/bin/myservice
      - uses: easingthemes/ssh-deploy@v4
        with:
          SSH_PRIVATE_KEY: ${{ secrets.DEPLOY_SSH_KEY }}
          REMOTE_HOST: ${{ secrets.DEPLOY_IP_ADDRESS }}
          REMOTE_USER: ${{ secrets.DEPLOY_USER }}
          SOURCE: ./deployment/systemd/
          TARGET: /home/${{ secrets.DEPLOY_USER }}/.config/systemd/user/
          ARGS: '--mkpath'
          SCRIPT_AFTER: |
            systemctl --user daemon-reload # Make sure systemd picks up the new Unit file
            systemctl --user enable myservice
            systemctl --user restart myservice
```

**Comment by easingthemes** [comment]
Fixed in the latest version v4.1.9 - https://github.com/easingthemes/ssh-deploy/pull/167/files#diff-9b163fdf32814f608c5eb67fb0841aa3dfa17545829fbbea2fe053b5991796e6R16.

**Comment by codespearhead** [comment]
@andsnpl Can you corroborate the fix and close the issue?
